storage/innobase/trx/trx0trx.cc

```diff
@@ -1237,7 +1237,7 @@ trx_finalize_for_fts(
 }
 
 extern "C" MYSQL_THD thd_increment_pending_ops(MYSQL_THD);
-extern "C" void  thd_decrement_pending_ops(MYSQL_THD);
+extern "C" void  thd_decrement_pending_ops(void*);
```

Comment:
This change is necessary for fixing runtime errors from `clang -fsanitize=undefined`, similar to ones fixed in #3322.

I paid attention to this, because we are changing the call in `trx_flush_log_if_needed()`. Even if #3490 is eventually completed and merged, the code in `trx_flush_log_if_needed()` will be different in this 10.11 branch and hence I think that we should actually correct this as part of this change.

Reply:
No, it is not necessarily to fix it exactly like this.

Reply:
Can you please leave  it as-is, and I will fix the overly picky compiler myself. We can't convert everything to void*, to please picky tools.

Reply:
If you add a wrapper function, it will likely incur some runtime overhead.

Reply:
I do not think so, not when it is trivial, static, inline, and only casts typed pointer parameter to void *. Compilers are smart.
Anyway it does not make any sense to convert everything to void*, because a tool incorrectly said so. This creates a WTF for the reader.